tailwind.config.js

```diff
@@ -91,6 +91,7 @@ module.exports = {
                 sm: "2px",
                 md: "6px",
                 lg: "10px",
+                "2xl": "16px",
```

Comment:
- I mean in this line and the other line, the quotation should be removed or should be added to the other configs.